src/set.cr

```diff
@@ -110,32 +195,48 @@ struct Set(T)
     end
   end
 
+  # Writes a string representation of the set to `io`
   def to_s(io)
     io << "Set{"
     join ", ", io, &.inspect(io)
     io << "}"
   end
 
+  # Returns `true` if the set is a subset of the `other` set
+  #
+  # This set must have fewer elements than the `other` set, and all of elements
+  # in this set must be present in the `other` set.
+  #
+  #     Set.new([1,5]).subset? Set.new([1,3,5]) # => true
   def subset?(other : Set)
```

Comment:
It looks like ruby has it as [`proper_subset`](http://ruby-doc.org/stdlib-2.2.3/libdoc/set/rdoc/Set.html#method-i-proper_subset-3F). I've not actually used these methods, but it probably does make sense to add them